### DISCUSSION TITLE
专业版账户设备无法解绑

### DISCUSSION THREAD

**Discussion by kessty** [getlantern/lantern-forum-cn#736] [Q&A]
@Mike777Khoo   专业版账户里面其中一台设备苹果电脑无法解绑，多次尝试无果。

![WX20240726-115639@2x](https://github.com/user-attachments/assets/00f5b50b-c79f-4f50-97b2-b310f1998a6b)

**Comment by MaxMKMax** [comment]
@kessty 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by kessty** [reply]
log/ticket号码：75745465

**Comment by MaxMKMax** [comment]
@kessty 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。谢谢。

**Comment by kessty** [comment]
报告早提交了，也没人跟进，就这服务真是无语…

| |
张伟
|
|
***@***.***
|
---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年07月29日 23:15 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | kessty ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 专业版账户设备无法解绑 (Discussion #736) |

@kessty 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。谢谢。

**Reply by MaxMKMax** [reply]
@kessty 我这里没权限也无法看系统里的报告. 我需要您提供我 8 位数票号. 我今晚会和他们开会才能谈. 不好意思了, 麻烦您啦!

**Comment by ycjdixon** [comment]
@kessty  这个BUG我也遇到过，你可以试试我的办法。你先在10.14.5那台电脑上蓝灯退出登录，然后再用10.15.7那台电脑上授权码登录，之后大概率就可以在10.15.7那台电脑上解绑了。这个问题也困扰了我很久，最后自己琢磨出来的办法。遇到问题还是自己动脑经想办法吧。他们服务跟不上的。7.8.6版本闪退问题到现在还没解决呢！

**Comment by kessty** [comment]
他们不是服务跟不上，是根本没有服务，iOS版本到现在也无人管，一提问就说马上发布新版本，从去年一直骗到今年… 就一皮包运营

| |
张伟
|
|
***@***.***
|
---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年07月30日 21:14 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | kessty ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 专业版账户设备无法解绑 (Discussion #736) |

@kessty 这个BUG我也遇到过，你可以试试我的办法。你先在10.14.5那台电脑上蓝灯退出登录，然后再用10.15.7那台电脑上授权码登录，之后大概率就可以在10.15.7那台电脑上解绑了。这个问题也困扰了我很久，最后自己琢磨出来的办法。遇到问题还是自己动脑经想办法吧。他们服务跟不上的。7.8.6版本闪退问题到现在还没解决呢！

**Reply by MaxMKMax** [reply]
@kessty 我们已经提交了7次新版本，但一直被IOS拒绝。不是我们一直在撒谎，而是问题是ios太严格了。我们不停地编辑、编辑和提交。但他们仍然没有批准最新版本。

**Comment by kessty** [comment]
我的账户下设备无法删除的问题什么时候能解决？客户端里提交了这么长时间，也没人搭理…

| |
张伟
|
|
***@***.***
|
---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年08月01日 01:24 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | kessty ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 专业版账户设备无法解绑 (Discussion #736) |

@kessty 我们已经提交了7次新版本，但一直被IOS拒绝。不是我们一直在撒谎，而是问题是ios太严格了。我们不停地编辑、编辑和提交。但他们仍然没有批准最新版本。
